### PR TITLE
Spell-check Jdbi documentation

### PR DESCRIPTION
**Pull request by sman-81**
An attempt to hunt down typos and idiosyncrasies in `index.adoc` (to avoid single typo PRs)

### CODE REVIEW

File: docs/src/adoc/index.adoc [hunk 1]
```diff
@@ -4900,12 +4900,12 @@ The H2 database is one of the most popular choices for embedded testing. The lin
...
+* link:{jdbidocs}/testing/junit5/JdbiPostgresExtension.html[JdbiPostgresExtension^] manages a local database on the host. It uses the link:https://pg-embedded.softwareforge.de/[pg-embedded] library that can spin up Postgres instances on various OS and architectures. This is the fastest way to get to a Postgres instance. Most of the Jdbi test classes that use Postgres use this extension.
 * link:{jdbidocs}/testing/junit5/JdbiExternalPostgresExtension.html[JdbiExternalPostgresExtension^] supports an external Postgres database and can run tests against any local or remote database.
 * link:{jdbidocs}/testing/junit5/JdbiOtjPostgresExtension.html[JdbiOtjPostgresExtension^] supports the link:https://github.com/opentable/otj-pg-embedded[OpenTable Java Embedded PostgreSQL component].
 
 
-=== Using Testcontainers
+=== Using Test Containers
```
Comment (hgschmie) [inline docs/src/adoc/index.adoc:4908]: The project is called "Testcontainers" (one word), this chapter is not about using containers for testing but that one specific project. So "Testcontainers" is correct.

File: docs/src/adoc/index.adoc [hunk 2]
```diff
@@ -5194,7 +5194,7 @@ Just tell us about your types by installing the plugin and configuring your `Imm
 
 `jdbi.getConfig(JdbiImmutables.class).registerImmutable(MyValueType.class)`
 
-The configuration will both register appropriate `RowMapper`&#8288;s as well as configure the new `bindPojo` (or `@BindPojo`) binders:
```
Comment (hgschmie) [inline docs/src/adoc/index.adoc:5197]: this is the "word joiner" unicode that pulls the `RowMapper` and the s back together.

File: docs/src/adoc/index.adoc [hunk 3]
```diff
@@ -6899,7 +6899,7 @@ link:{jdkdocs}/java/lang/reflect/Type.html[java.lang.reflect.Type^] object used
 to represent generics in Java.
 
 
-==== GenericTypes
+==== Generic Types
```
Comment (hgschmie) [inline docs/src/adoc/index.adoc:6902]: this should be "The GenericTypes helper". It references a specific class. Don't split that.

**Review by hgschmie — CHANGES_REQUESTED** [review]
mostly fine, some minor points

File: docs/src/adoc/index.adoc (hunk 1 above)
Comment (sman-81) [reply docs/src/adoc/index.adoc:4908]: ah yes of course

File: docs/src/adoc/index.adoc (hunk 2 above)
Comment (sman-81) [reply docs/src/adoc/index.adoc:5197]: cleeeeeever :)

**Comment by sman-81** [comment]
Thanks for your feedback @hgschmie
PR updated (squashed and force-pushed)

**Review by hgschmie — APPROVED** [review]

**Comment by hgschmie** [comment]
image

**Comment by sman-81** [comment]
git-merge

**Pull request merged**